.buildkite/pipeline.yml

```
@@ -65,11 +65,11 @@ steps:
       --no-out-link
       --argstr scenario './result/bin/homebase-lite-test --cleveland-mode=only-network')
 
-  - label: scheduled kathmandu test
+  - label: scheduled lima test
```

nitpick, but it would probably be more correct to name this:
```suggestion
  - label: scheduled limanet test
```

---

Okay, done. I've amended the last commit instead of doing the whole fixup ritual, because the change is trivial.